app/routes/explore.js

```diff
@@ -76,6 +76,14 @@ export default Route.extend({
         val  : `%${params.location}%`
       });
     }
+    if (params.cfs) {
+      let cfsVal = params.cfs === 'Open' ? true : false;
+      filterOptions.push({
+        name : 'is_sessions_speakers_enabled',
+        op   : 'eq',
+        val  : cfsVal
```

Comment:
This can just be `params.cfs === 'open'`

Reply:
@Anupam-dagar You missed this one!

Reply:
@kushthedude this has also been fixed, check in changes section.